modules/microshift-updates-troubleshooting.adoc

```diff
@@ -8,13 +8,13 @@
 
 In some cases, {microshift-short} might fail to update. In these events, it is helpful to understand failure types and how to troubleshoot them.
 
-[id="microshift-update-path-blocked-by-version-sequence_{context}"]
-== Update path is blocked by {microshift-short} version sequence
-{microshift-short} requires serial updates. Attempting to update {microshift-short} by skipping a minor version fails:
+//[id="microshift-update-path-blocked-by-version-sequence_{context}"]
+//== Update path is blocked by {microshift-short} version sequence
+//Certain versions of {microshift-short} require serial updates. Attempting to update {microshift-short} by skipping a minor version fails:
 
-* For example, if your current version is `4.14.5`, but you try to update from that version to `4.16.0`, the message, `executable (4.16.0) is too recent compared to existing data (4.14.5): version difference is 2, maximum allowed difference is 1` appears and {microshift-short} fails to start.
+//* For example, if your current version is `4.14.5`, but you try to update from that version to `4.16.0`, the message, `executable (4.16.0) is too recent compared to existing data (4.14.5): version difference is 2, maximum allowed difference is 1` appears and {microshift-short} fails to start.
```

Comment:
I believe this is not a valid statement anymore. MicroShift can upgrade Y-2

Reply:
Right, hence it is commented out and not in the doc preview: https://76585--ocpdocs-pr.netlify.app/microshift/latest/microshift_troubleshooting/microshift-troubleshoot-updates.html

Reply:
ah, that is what the `\\` is for. I couldn't find it in the document, and was confused. Sorry for my confusion.